docs/admin/install/serving/install-serving-with-yaml.md

```diff
@@ -219,106 +178,7 @@ Follow the procedure for the networking layer of your choice:
-    or CNAME from setting up networking, and configure it with your DNS provider as
-    follows:
-
-    1. If the networking layer produced an External IP address, then configure a
-      wildcard `A` record for the domain by running the command:
-
-        ```bash
-        *.<domain-suffix> == A 35.233.41.212
-        ```
-        Where `<domain-suffix>` is the domain suffix for your cluster
-
-    1. If the networking layer produced a CNAME, then configure a CNAME record for the domain by running the command:
-
-        ```bash
-        *.<domain-suffix> == CNAME a317a278525d111e89f272a164fd35fb-1510370581.eu-central-1.elb.amazonaws.com
-        ```
-        Where `<domain-suffix>` is the domain suffix for your cluster
-
-    1. After your DNS provider is configured, direct Knative to use that domain by running the command:
-
-        ```bash
-        kubectl patch configmap/config-domain \
-          --namespace knative-serving \
-          --type merge \
-          --patch '{"data":{"<domain-suffix>":""}}'
-        ```
-        Where `<domain-suffix>` is the domain suffix for your cluster
-
-=== "Temporary DNS"
-
-    If you are using curl to access the sample applications or your own Knative app, and are
-    unable to use the Magic DNS (xip.io) or Real DNS methods, you can use a temporary approach.
-
-    See the Real DNS method for a permanent solution.
-
-    Temporary DNS is useful for evaluating Knative without altering the DNS configuration, as in the
-    Real DNS method, or cannot use the Magic DNS method due to using, for example, minikube locally
-    or IPv6 clusters.
-
-    To access your application using curl using this method:
-
-    1. After starting your application, get the URL of your application by running the command:
-
-        ```bash
-        kubectl get ksvc
-        ```
-
-        !!! success "Verify the output"
-
-            ```bash
-            NAME            URL                                        LATESTCREATED         LATESTREADY           READY   REASON
-            helloworld-go   http://helloworld-go.default.example.com   helloworld-go-vqjlf   helloworld-go-vqjlf   True
-            ```
-
-    1. Instruct curl to connect to the External IP address or CNAME defined by the
-      networking layer in section 3 above, and use the `-H "Host:"` command-line
-      option to specify the Knative application's hostname.
-
-      For example, if the networking layer defines your External IP address and port to be
-      `http://192.168.39.228:32198` and you wish to access the above `helloworld-go` application,
-      run the command:
-
-        ```bash
-        curl -H "Host: helloworld-go.default.example.com" http://192.168.39.228:32198
-        ```
-
-        !!! success "Verify the output"
-            In the case of the provided `helloworld-go` sample application, when using the default configuration the output is:
-
-            ```
-            Hello Go Sample v1!
-            ```
+{% include "dns.md" %}
```

Comment:
```suggestion
<!-- This is a snippet, to find out more about Snippets, and to see existing Snippets, see: /docs/snippets-->
--8<--  "dns.md"
```
Aligning around existing Snippets plugin which is referenced in snippets/README.md

Reply:
There's a problem with including that DNS snippet, here.

It is **only** valid if you install Knative with Operator. It advises changing KnativeServing resource (in _Real DNS_ tab). When doing installation via YAML, like this page does, this isn't valid as then there's no KnativeServing resource.

In case of installing via YAML, we need to update config map directly, like this:

```bash
kubectl patch configmap/config-domain \
  --namespace knative-serving \
  --type merge \
  --patch '{"data":{"<domain-suffix>":""}}'
```

Reply:
oh what a pain! I guess I goofed on this when I pulled this out into a snippet :(. Will have to split it in to smaller snippets I guess so operator can have its own Real DNS tab, tho I suggest we worry about that in another PR.

@omerbensaadon unfortunately this snippet would need to be `{%` format because it uses the artefact macro inside it so has to be included via the macro include syntax. Possibly we should actually standardise on that instead of `--8<--`.

Reply:
@julz Alright, I can agree to do it in the next PR, but take care of it, please.

I'm commenting on this exact problem for the 2nd time already (Real DNS tab).

Reply:
I mean let's worry about moving this back to snippets in a follow-on PR (so that that can be my problem instead of @abrennan89's since it was me who messed up the snippet). IMO we should fix the text to be correct here (and I'll make sure not to break it next time!)

Reply:
@julz this confuses me but I'll throw it on the agenda, would be ideal if we standardized on an extension with detailed documentation...

Reply:
@julz can you open this issue? I would but I don't think I am fully grasping the context here

Reply:
I should have blocked this PR, on @julz comment. Now, it's merged and the issue persist.

Reply:
Oops yeah, I guess I'll fix it all in one go. Will try to PR today

Reply:
@cardil I figured this is something that could be fixed in a follow-on PR... was that a mistake?